src/Test/Utilities/Portable/Compilation/TestOperationVisitor.cs

```diff
@@ -44,26 +44,17 @@ public override void Visit(IOperation operation)
                 var type = operation.Type;
                 var constantValue = operation.ConstantValue;
 
-                // operation.Language can throw due to https://github.com/dotnet/roslyn/issues/23819
+                Assert.NotNull(syntax);
```

Comment:
>Assert.NotNull(syntax); [](start = 16, length = 23)

Would it be possible to add similar assert to the other visitor we are using for tests?

Reply:
This is the PR to fix that issue :)
I'll add a test to this PR.

---
In reply to: [197882035](https://github.com/dotnet/roslyn/pull/28084#discussion_r197882035) [](ancestors = 197882035)